### PR TITLE
Move ci from travis to github actions

### PR DESCRIPTION
**Pull request by Umekawa**
- move ci from travis to github action

### CODE REVIEW

File: lib/js_rails_routes/configuration.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Configuration
                   :route_filter,
                   :route_set_filter
 
-    def initialize
+    def initialize # rubocop:disable Metrics/AbcSize
```
Comment (Umekawa) [inline lib/js_rails_routes/configuration.rb:16]: These methods seem to be better without method splitting.

**Comment by Umekawa** [comment]
Now, codeclimate is not support rubocop of 1.27.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # rake js:routes
 
 [![Gem](https://img.shields.io/gem/v/js_rails_routes.svg?maxAge=2592000)](https://rubygems.org/gems/js_rails_routes)
-[![Build Status](https://travis-ci.org/increments/js_rails_routes.svg?branch=master)](https://travis-ci.org/increments/js_rails_routes)
+![Build Status](https://github.com/increments/js_rails_routes/actions/workflows/test.yml/badge.svg?branch=master)
```
Comment (Umekawa) [inline README.md:4]: Now, master branch does not have build result.
For check result, see result of this branch.
https://github.com/increments/js_rails_routes/actions/workflows/test.yml/badge.svg?branch=move-github-actions

**Comment by Umekawa** [comment]
[This test](https://github.com/increments/js_rails_routes/runs/6364550065?check_suite_focus=true) has failed, but is not covered in this PR.

**Review by mziyut — APPROVED** [review]
✅ GTM!

**Pull request merged**